configs/openeo-cloud.json

```diff
@@ -1,30 +1,147 @@
 {
   "index_name": "openeo-cloud",
   "start_urls": [
-    "https://docs.openeo.cloud"
+    {
+      "url": "https://open-eo.github.io/openeo-js-client/latest/",
+      "selectors_key": "js-client",
+      "page_rank": 0
+    },
+    {
+      "url": "https://open-eo.github.io/openeo-r-client/",
+      "selectors_key": "r-client",
+      "page_rank": 0
+    },
+    {
+      "url": "https://open-eo.github.io/openeo-python-client/",
+      "selectors_key": "python-client",
+      "page_rank": 0
+    },
+    {
+      "url": "https://docs.openeo.cloud",
+      "page_rank": 1
+    },
```

Comment:
Broader paths should be at the end of the `start_urls` array otherwise they will get picked up before deeper ones

Reply:
That makes a lot of sense, fixed! Thanks.